packaging/linux/common/khiops

```diff
@@ -70,13 +61,35 @@ fi
 export CLASSPATH=$CLASSPATH:$KHIOPS_CLASSPATH
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$KHIOPS_JAVA_PATH
 
-if [[  $# -eq 0  ]]; then
-    # run without parameters
-    # run and save scenario and log files in directory KHIOPS_LAST_RUN_DIR
-    $KHIOPS_MPI_COMMAND MODL -o "${KHIOPS_LAST_RUN_DIR}"/scenario._kh -e "${KHIOPS_LAST_RUN_DIR}"/log.txt
+launch_khiops() {
+    if [[ $# -eq 0 ]]; then
+        # run without parameters
+        # run and save scenario and log files in directory KHIOPS_LAST_RUN_DIR
+        $KHIOPS_MPI_COMMAND "$KHIOPS_PATH -o ${KHIOPS_LAST_RUN_DIR}/scenario._kh -e ${KHIOPS_LAST_RUN_DIR}/log.txt"
```

Comment:
The enclosing quotes should be removed around the arguments to `$KHIOPS_MPI_COMMAND`; hence, this line should be:
```bash
 $KHIOPS_MPI_COMMAND $KHIOPS_PATH -o ${KHIOPS_LAST_RUN_DIR}/scenario._kh -e ${KHIOPS_LAST_RUN_DIR}/log.txt
```